lib/index.js

```diff
@@ -342,7 +353,44 @@ module.exports = (function() {
                         self.moveAllImages(drawing, row.attrib.r, newTableRows.length)
                     }
                 }				
+                
+                //Filtre all the cellsForsubstituteTable cell with the 'row' cell
+                //Don't can make a filter instead of this ugly foreach ?
```

Comment:
OK.
Would you rather I do a for loop `for (const rowCell of row.findall("c"))` rather than a foreach `row.findall("c").forEach(function(rowCell) {`?
Or do we leave it like that (with a foreach)?

Reply:
I just succeeded to do my filter.
This simplifies the code and its reading I find.